src/core/custom_tex_cache.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Citra Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include <stdexcept>
+#include <vector>
+#include "common/common_types.h"
+#include "custom_tex_cache.h"
+
+namespace Core {
+CustomTexCache::CustomTexCache() = default;
+
+CustomTexCache::~CustomTexCache() = default;
+
+bool CustomTexCache::IsTextureDumped(u64 hash) const {
+    return dumped_textures.find(hash) != dumped_textures.end();
```

Comment:
```suggestion
    return dumped_textures.count(hash);
```

Also... you are actually using this as if it is an `unordered_set` (just querying the existence of the key without caring about the value which is always `true` anyway)